### PR TITLE
Update zh-docs with 4865ddf(v0.14.3+)

### PR DESCRIPTION
**Pull request by iamchenxin**
update zh-docs to current master

### CODE REVIEW

File: docs/docs/10.10-shallow-compare.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ export class SampleComponent extends React.Component {
 }
 ```
 
-`shallowCompare` performs a shallow equality check on the current `props` and `nextProps` objects as well as the current `state` and `nextState` objects.  
+`shallowCompare` performs a shallow equality check on the current `props` and `nextProps` objects as well as the current `state` and `nextState` objects.
```
Comment (zpao) [inline docs/docs/10.10-shallow-compare.md]: Please revert this and the other change here. It looks intentional, though it's still not great formatting when rendered (I'd rather leave it and fix separately). I'm guessing this is just a result of saving in your editor. I encourage you to install a plugin that supports the editorconfig file we have which overrides that built-in behavior (http://editorconfig.org/#download)

File: docs/docs/getting-started.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 id: getting-started
 title: Getting Started
+permalink: getting-started.html
```
Comment (zpao) [inline docs/docs/getting-started.md]: This is unnecessary for the en-US pages.

File: docs/docs/ref-01-top-level-api.md [hunk 3]
```diff
@@ -66,9 +66,7 @@ factoryFunction createFactory(
 )
 ```
 
-Return a function that produces ReactElements of a given type. Like `React.createElement`,
-the type argument can be either an html tag name string (eg. 'div', 'span', etc), or a
-`ReactClass`.
+Return a function that produces ReactElements of a given type. Like `React.createElement`,the type argument can be either an html tag name string (eg. 'div', 'span', etc), or a `ReactClass`.
```
Comment (zpao) [inline docs/docs/ref-01-top-level-api.md]: Please either revert or ensure that when you delete the newline, you put a space in (you need one after the comma).

File: docs/docs/videos.md [hunk 4]
```diff
@@ -85,7 +85,7 @@ A [tagtree.tv](http://tagtree.tv/) video conveying the principles of [Thinking i
 19:36 – DOM Manipulation
 </td><td>
 23:06 – Supporting node.js<br />
-24:03 – rendr<br />
+24:03 – render<br />
```
Comment (zpao) [inline docs/docs/videos.md]: This should be "rendr" - it's in reference to https://github.com/rendrjs/rendr

File: docs/docs/ref-01-top-level-api.md (hunk 3 above)
Comment (iamchenxin) [inline docs/docs/ref-01-top-level-api.md]: i will do a revert, and separate en-docs and zh-docs commit.
for  en-docs ,the mainly modification were delete the unnecessary newline

File: docs/docs/10.10-shallow-compare.md (hunk 1 above)
Comment (iamchenxin) [inline docs/docs/10.10-shallow-compare.md]: ok

**Comment by iamchenxin** [comment]
i reset the branch ,keep all of the zh-docs in the branch.
and i should push a new branch for delete newlines in en-docs, to keep translate-docs consistence with en-docs

**Comment by iamchenxin** [comment]
@zpao should i open another branch for this, or a reset is just ok ?

**Comment by zpao** [comment]
No, this is fine. Sorry for not getting back to you when you first reset!

**Pull request merged**